Event/SendinblueListSubscriber.php

```diff
@@ -36,12 +37,19 @@ class SendinblueListSubscriber implements EventSubscriberInterface
      */
     private $contactsApi;
 
+    /**
+     * @var LinkProviderPoolInterface
+     */
+    private $linkProviderPool;
+
     public function __construct(
         RequestStack $requestStack,
+        LinkProviderPoolInterface $linkProviderPool,
```

Comment:
@alexander-schranz can we do the BC break here or should we add a BC layer?

Reply:
as it is not much work a bc layer should be added here. includes that this need to be the the last parameter here and be ` = null` by default